frontend/src/shared/actions.js

```diff
@@ -3,6 +3,8 @@ import { push } from 'connected-react-router'
 
 export const LOGIN_USER = 'LOGIN_USER'
 export const LOGOUT_USER = 'LOGOUT_USER'
+export const JUDGE_ROLE = 'JUDGE_ROLE'
+export const ADMIN_ROLE = 'ADMIN_ROLE'
```

Comment:
I think I would prefer these actions to be named:

```js
export const SWITCH_TO_JUDGE = 'SWITCH_TO_JUDGE'
export const SWITCH_TO_ADMIN = 'SWITCH_TO_ADMIN'
```

Don't forget to update the references in other files.